ci/pipeline

```diff
@@ -229,9 +229,6 @@ def master(): Unit = {
 
   // Uploads
   val maybeArtifact = uploadTarballPackagesToS3(version, s"builds/$version")
-  maybeArtifact.foreach { artifact =>
```

Comment:
This code shouldn't be called for non-master branches at all

Reply:
but we want to create the package for non-master branches...

Reply:
I'd rather extend https://github.com/mesosphere/marathon/pull/6421/files#diff-60758bba2cfe833f99a8029227ae4507L276
```
 def jenkins(): Unit = {	
   utils.request() match {	
     case PR(pullId) => pr(pullId)
     case Release(version) => release(version) // This, does the same as master() but don't patch DC/OS
     case Master => master()
   }
 }
```